src/App.jsx

```diff
@@ -20,21 +20,17 @@ function App() {
     <Router>
       <ExternalNavigationBar />
       <Routes>
-        <Route path="/Home" element={<HomePage />}>
-          {' '}
-        </Route>
-        <Route path="/Home/howto" element={<HowToPage />}>
-          {' '}
-        </Route>
-        <Route path="/Home/schoolnav" element={<SchoolNavPage />}>
-          {' '}
-        </Route>
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/Home" element={<HomePage />}/>
+        <Route path="/Home/howto" element={<HowToPage />}/>
+        <Route path="/Home/schoolnav" element={<SchoolNavPage />}/>
+        <Route path="/Home/signin" element={<LoginPage />}/>
+        <Route path="/Home/signup" element={<CreateAccount />}/>
       </Routes>
 
       <Providers>
         <PageWrapper>
           <Routes>
-            <Route path="/" element={<LoginPage />} />
```

Comment:
WE delete this because we added it to the external nav bar?

Reply:
We delete this because the default home page"/" should be set to "/Home."